internal/auth0/page/resource_test.go

```diff
@@ -0,0 +1,127 @@
+package page_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/auth0/terraform-provider-auth0/internal/acctest"
+)
+
+const testAccPagesCreate = `
```

Comment:
Perhaps consider adding a test case where we only modify a subset of the inner blocks?